### PR TITLE
[1] Wiki Clean Up: Addresses Some Stales and Isolated Issues

### PR DESCRIPTION
**Pull request by salmad3**
Mainly closes outstanding stales.

### CODE REVIEW

**Comment by salmad3** [comment]
Perhaps it's better to knock off some issues first and not over do the reviewing. Will open a similar PR addressing others.

File: docs/kusama-index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Kusama
 sidebar_label: Home
 ---
 
-Kusama is a "canary network" for Polkadot, an early unaudited release of the code that is available
+Kusama is a "canary network" for Polkadot; an early, unaudited release of the code that is available
```
Comment (Swader) [inline docs/kusama-index.md]: No longer unaudited

**Review by Swader — APPROVED** [review]
Ok to merge after addressing comment

File: docs/kusama-index.md (hunk 1 above)
Comment (salmad3) [reply docs/kusama-index.md]: updated

**Comment by salmad3** [comment]
> Ok to merge after addressing comment

Updated and merging.

**Pull request merged**